packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/ios/runsdkapp.md

```diff
@@ -1,12 +1,9 @@
 
-1. If not already done, set up your Okta org by completing these steps: [Set up your Okta org for a password factor only use case](/docs/guides/oie-embedded-common-org-setup/ios/main/#set-up-your-okta-org-for-a-password-factor-only-use-case).
-1. If not already done,
-   [download and set up the sample app](/docs/guides/oie-embedded-common-download-setup-app/ios/main/).
-1. Using Xcode, locate and open the `okta-idx.xcworkspace` file in the following path: `okta-idx-swift/`
-1. Optionally, open the `Okta.plist` file and configure it with the settings from your org. See
-   the [Okta.plist](/docs/guides/oie-embedded-common-download-setup-app/ios/main/#okta-plist) section for more information.
+1. If you haven't already done so, [set up your Okta org](/docs/guides/oie-embedded-common-org-setup/ios/main/#set-up-your-okta-org-for-a-password-factor-only-use-case).
+1. If you haven't already done so, [download and set up the sample app](/docs/guides/oie-embedded-common-download-setup-app/ios/main/).
+1. Using Xcode, locate and open the `okta-idx.xcworkspace` file in the following path: `okta-idx-swift/`.
+1. Optionally, open the `Okta.plist` file and configure it with the settings from your org. See [Okta.plist](/docs/guides/oie-embedded-common-download-setup-app/ios/main/#okta-plist) for more information.
 1. Click **Run** to launch the application in the iOS simulator.
-1. If you didn't didn't configure `Okta.plist`, you can add your org settings to the Configuration
-   screen by tapping the **Configure** link on the startup screen. See the
-   [Configuration screen](/docs/guides/oie-embedded-common-download-setup-app/ios/main/#configuration-page) section for more information.
+1. If you didn't didn't configure the `Okta.plist` file, you can add your org settings to the Configuration
+   page by tapping the **Configure** link on the startup screen. See [Configuration page](/docs/guides/oie-embedded-common-download-setup-app/ios/main/#configuration-page) for more information.
```

Comment:
startup screen or start-up screen?

Reply:
I'm not sure if "startup" screen is a standard iOS term. I'll go with "start-up" since it's used as an adjective here.